modules/app-configuration/configuration-store/main.bicep

```diff
@@ -82,12 +79,12 @@ param privateEndpoints privateEndpointType
 
 var enableReferencedModulesTelemetry = false
 
-var identityType = systemAssignedIdentity ? 'SystemAssigned' : !empty(userAssignedIdentities) ? 'UserAssigned' : 'None'
+var formattedUserAssignedIdentities = reduce(map((managedIdentities.?userAssignedResourcesIds ?? []), (id) => { '${id}': {} }), {}, (cur, next) => union(cur, next)) // Converts the flat array to an object like { '${id1}': {}, '${id2}': {} }
 
-var identity = {
-  type: identityType
-  userAssignedIdentities: !empty(userAssignedIdentities) ? userAssignedIdentities : null
-}
+var identity = !empty(managedIdentities) ? {
```

Comment:
As per original template this may need to be only one or the other

Reply:
Of so, we should probably mention this in the description

Reply:
Contrary to the previous implementation, the resource supports any combination of identities ([see template reference](https://learn.microsoft.com/en-us/azure/templates/microsoft.appconfiguration/configurationstores?pivots=deployment-language-bicep#resourceidentity)), so I changed the implementation accordingly.